draft-terbu-sd-jwt-vc.md

```diff
@@ -524,68 +524,108 @@ model defined in [@VC-DATA], this specification defines a mapping algorithm for
 VC-SD-JWT and VP-SD-JWT to the vocabulary and data model defined W3C VCDM 2.0
 which is based on JSON-LD.
 
-### VC Directory
+### W3C VC Specifications Directory
 
 This specification registers the media type `application/vc+sd-jwt` in the
-W3C Verifiable Credentials (VC) Directory.
+W3C Verifiable Credentials (VC) Directory [@VC-DIR].
 
 ### Mapping Algorithm
 
 The following is a uni-directional transformation algorithm that takes in a
 VC-SD-JWT conformant to this specification and maps it
-to the corresponding properties in the W3C VCDM 2.0 [VCDM2.0]
+to the corresponding properties in the W3C VCDM 2.0 [@VC-DATA]
 which is based on a JSON-LD vocabulary. It includes specific handling for JWT
 claims used in this specification. The function returns a Verifiable
 Credential object in the W3C VCDM 2.0 format.
 
+Procedure:
+1. Let `payload` be the unsecured payload of the VC-SD-JWT reconstructed from the SD-JWT and Disclosures.
+1. Let `VC` be an empty JSON object that represents the transformed Verifiable Credential:
+  - Set the `@context` property of `VC` to `"https://www.w3.org/ns/credentials/v2"`.
```

Comment:
```suggestion
1. Let `vc` be an empty JSON object that represents the transformed Verifiable Credential:
  - Set the `@context` property of `vc` to `"https://www.w3.org/ns/credentials/v2"`.
```

why uppercase..?

Reply:
@Sakurann `VC` is a variable that contains a JSON object in that case which is also the resulting W3C VC. I made it lower-case now. Does that work for you?

Reply:
We will use italic instead of double quotes for variable names.